modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -51,7 +51,7 @@ def show
 
       def fetch_errored(claim)
         if claim.evss_response&.any?
-          errors = format_evss_errors(claim.evss_response['messages'])
+          errors = format_evss_errors(claim.evss_response)
```

Comment:
`evss_response` is of type array. Trying to access a `messages` key was causing a type error because it's not present on `evss_response`